stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -284,14 +287,14 @@ def test_potential_qty(self):
             "Receiving variant 2's component should not change "
             "variant 1's potential")
         self.assertPotentialQty(
-            self.var2, 42.0,
+            self.var2, 313.0,
```

Comment:
@grindtildeath The Tests depended on Demodata of product, mrp. etc which has changed or was removed since v9. So i tried to recreate the data. Here there probably is a product_qty missing. Since its Likely that the original bom line had a efficiency on it, I ditdn't try to mach it exactly.